docs/vision-usage.rst

```diff
@@ -30,76 +30,15 @@ or pass in ``credentials`` and ``project`` explicitly
      >>> from google.cloud import vision
      >>> client = vision.Client(project='my-project', credentials=creds)
 
-Annotating an Image
--------------------
-
-Annotate a single image
-~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code-block:: python
-
-    >>> import io
-    >>> from google.cloud import vision
-    >>> client = vision.Client()
-    >>> with io.open('./image.png', 'rb') as image_file:
-    ...     image = client.image(content=image_file.read())
-    >>> faces = image.detect_faces(limit=10)
-    >>> faces[0].landmarks.left_eye.position.x_coordinate
-    ... 1004.8003
-
-Annotate multiple images
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code-block:: python
-
-    >>> import io
-    >>> from google.cloud import vision
-    >>> client = vision.Client()
-    >>> with io.open('./image.png', 'rb') as image_file:
-    ...     image_one = client.image(content=image_file.read())
-    >>> image_two = client.image(source_uri='gs://my-storage-bucket/image.jpg')
-    >>> with client.batch():
-    ...     labels = image_one.detect_labels()
-    ...     faces = image_two.detect_faces(limit=10)
-
-No results returned
-~~~~~~~~~~~~~~~~~~~
-
-Failing annotations return no results for the feature type requested.
-
-.. code-block:: python
-
-    >>> from google.cloud import vision
-    >>> client = vision.Client()
-    >>> image = client.image(source_uri='gs://my-storage-bucket/image.jpg')
-    >>> logos = image.detect_logos(limit=10)
-    >>> logos
-    []
-
-
-Manual Detection
-~~~~~~~~~~~~~~~~
-
-You can call the detection method manually.
-
-.. code-block:: python
-
-    >>> from google.cloud import vision
-    >>> from google.cloud.vision.image import Feature
-    >>> from google.cloud.vision.image import FeatureTypes
-    >>> client = vision.Client()
-    >>> image = client.image(source_uri='gs://my-test-bucket/image.jpg')
-    >>> features = [Feature(FeatureTypes.FACE_DETECTION, 5),
-    ...             Feature(FeatureTypes.LOGO_DETECTION, 3)]
-    >>> annotations = image.detect(features)
```

Comment:
Why this big deletion? (I see "no results" has moved down, but the rest seems to be just gone?)